stories/location.NMNH.mdx

```diff
@@ -13,7 +13,7 @@ Address
 10th St. & Constitution Ave. NW, Washington, DC 20560
 Admission is free
 "
-external:
+asLink:
```

Comment:
@agurvich  we changed `external` to `asLink`!

Reply:
Question: What is the asLink property?

Reply:
as link property signals that Story will work as a link. More details in : https://github.com/NASA-IMPACT/veda-ui/issues/728, https://github.com/NASA-IMPACT/veda-ui/pull/735